dashboards/grafana-dashboard-MPC.configmap.yaml

```
@@ -0,0 +1,2517 @@
+              "mode": "single",
+              "sort": "none"
+            }
+          },
+          "pluginVersion": "11.6.3",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "P22466E8E7855F1E0"
+              },
+              "editorMode": "code",
+              "expr": "sum(kube_pod_status_phase{namespace=\"multi-platform-controller\", pod=~\"multi-platform-otp.*\", source_cluster=~\"$cluster\"})\nby (source_cluster)\n-\nsum(kube_pod_status_phase{namespace=\"multi-platform-controller\", phase=\"Running\", pod=~\"multi-platform-otp.*\", source_cluster=~\"$cluster\"})\nby (source_cluster)",
+              "legendFormat": "__auto",
+              "range": true,
+              "refId": "A"
+            }
+          ],
+          "title": "Non-Running OTP Pods Per Cluster",
+          "type": "timeseries"
+        },
+        {
+          "collapsed": false,
+          "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 19
+          },
+          "id": 18,
+          "panels": [],
+          "title": "Running Tasks",
+          "type": "row"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "P22466E8E7855F1E0"
+          },
+          "description": "The number of currently running tasks on each platform",
+          "fieldConfig": {
+            "defaults": {
+              "color": {
+                "mode": "thresholds"
+              },
+              "mappings": [],
+              "thresholds": {
+                "mode": "absolute",
+                "steps": [
+                  {
+                    "color": "green"
+                  },
+                  {
+                    "color": "red",
+                    "value": 80
+                  }
+                ]
+              }
+            },
+            "overrides": []
+          },
+          "gridPos": {
+            "h": 14,
+            "w": 22,
+            "x": 1,
+            "y": 20
+          },
+          "id": 19,
+          "options": {
+            "minVizHeight": 75,
+            "minVizWidth": 75,
+            "orientation": "auto",
+            "reduceOptions": {
+              "calcs": [
+                "lastNotNull"
+              ],
+              "fields": "",
+              "values": false
+            },
+            "showThresholdLabels": false,
+            "showThresholdMarkers": true,
+            "sizing": "auto",
+            "text": {
+              "titleSize": 8
+            }
+          },
+          "pluginVersion": "11.6.3",
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "P22466E8E7855F1E0"
+              },
+              "editorMode": "code",
+              "expr": "sum by(platform) (multi_platform_controller_running_tasks{source_cluster=~\"$cluster\", platform=~\"$platform\"})",
```

Not sure why those are exported as negative values, maybe it's making sense and I will stand corrected, but just felt like throwing it inhere for clarification - why is that if you happen to know and if it's supposed to be like that, feel free to ignore :) 
```suggestion
              "expr": "abs(sum by(platform) (multi_platform_controller_running_tasks{source_cluster=~\"$cluster\", platform=~\"$platform\"}))",
```
If you happen to change that, don't forget to update the thresholds for colors

---

It's an open bug so I want to leave it as it is, that metric isn't working well right now, that is why it's also in the "Invalid Metrics" section :)